docker/api/swarm.py

```diff
@@ -82,6 +83,7 @@ def get_unlock_key(self):
 
     @utils.minimum_version('1.24')
     def init_swarm(self, advertise_addr=None, listen_addr='0.0.0.0:2377',
+                   default_addr_pool=None, subnet_size=None,
```

Comment:
Should add new parameters at the end of the list so that callers of `init_swarm` that use ordered (i.e.: unnamed) parameters continue to work as expected.

e.g.: An existing function could call:
```python
s.init_swarm('192.168.0.1', '0.0.0.0:2377', True)
```

Which would now break.